python/cudf/cudf/core/frame.py

```diff
@@ -238,6 +239,66 @@ def _quantiles(
         result._copy_categories(self)
         return result
 
+    def repeat(self, repeats, axis=None):
+        assert axis in (0, None)
+        """Repeats elements consecutively
+
+        Parameters
+        ----------
+        repeats : int, array, numpy array, or Column
+            the number of times to repeat each element
+
+        Example
+        -------
+        >>> import cudf as cudf
+        >>> s = cudf.Series([0, 2]) # or DataFrame
+        >>> s
+        0    0
+        1    2
+        dtype: int64
+        >>> s.repeat([3, 4])
+        0    0
+        0    0
+        0    0
+        1    2
+        1    2
+        1    2
+        1    2
+        dtype: int64
+        >>> s.repeat(2)
+        0    0
+        0    0
+        1    2
+        1    2
+        dtype: int64
+        >>>
+        """
```

Comment:
Docstring needs to be moved above assertion 😄
```suggestion
        """Repeats elements consecutively

        Parameters
        ----------
        repeats : int, array, numpy array, or Column
            the number of times to repeat each element

        Example
        -------
        >>> import cudf as cudf
        >>> s = cudf.Series([0, 2]) # or DataFrame
        >>> s
        0    0
        1    2
        dtype: int64
        >>> s.repeat([3, 4])
        0    0
        0    0
        0    0
        1    2
        1    2
        1    2
        1    2
        dtype: int64
        >>> s.repeat(2)
        0    0
        0    0
        1    2
        1    2
        dtype: int64
        >>>
        """
        assert axis in (0, None)
```

Reply:
:1st_place_medal:  Na, it's cool. :man_facepalming: